src/components/Toaster/Toast/Toast.scss

```diff
@@ -14,20 +14,26 @@ $block: '.#{variables.$ns}toast';
     $containerClass: #{&}__container;
     $iconClass: #{&}__icon;
 
-    --yc-toaster-padding: 16px 16px 16px 48px;
-
+    display: flex;
+    flex-flow: row nowrap;
     box-sizing: border-box;
     position: absolute;
+    opacity: 0;
     width: inherit;
     margin-bottom: 10px;
-    padding: var(--yc-toaster-padding);
```

Comment:
Why is var removed?

Reply:
In the old layout, the size of the icon was implicitly declared in the padding and this var was redefined for the default toast (without icon).
Now icon has own container and padding is always `16px`.